cmd/tools/cli/integration_test.go

```diff
@@ -0,0 +1,48 @@
+// +build integration
+
+package cli
+
+import (
+	"bytes"
+	"strings"
+	"testing"
+
+	"github.com/spf13/cobra"
+)
+
+func setUp() (*cobra.Command, *bytes.Buffer) {
+	cmd := &cobra.Command{
+		Use:           "k3ai-cli-test",
+		SilenceErrors: true,
+		SilenceUsage:  true,
+	}
+	out := bytes.NewBuffer(nil)
+	cmd.SetOut(out)
+	cmd.SetErr(out)
+	setupCli(cmd)
+	return cmd, out
+}
+func TestApply(t *testing.T) {
+	cmd, out := setUp()
+	cmd.SetArgs([]string{"apply", "--kubectl", "argo"})
```

Comment:
```suggestion
	cmd.SetArgs([]string{"apply", "--kubectl", "ci-tests"})
```
I am going to change the name for `argo` 
for the tests, there is `https://github.com/kf5i/k3ai-plugins/tree/main/core/plugins/ci-tests`